scene/2d/camera_2d.cpp

```diff
@@ -754,6 +794,8 @@ void Camera2D::_bind_methods() {
 	ADD_GROUP("Smoothing", "smoothing_");
 	ADD_PROPERTY(PropertyInfo(Variant::BOOL, "smoothing_enabled"), "set_enable_follow_smoothing", "is_follow_smoothing_enabled");
 	ADD_PROPERTY(PropertyInfo(Variant::FLOAT, "smoothing_speed"), "set_follow_smoothing", "get_follow_smoothing");
+	ADD_PROPERTY(PropertyInfo(Variant::BOOL, "smoothing_rotation_enabled"), "set_enable_rotation_smoothing", "is_rotation_smoothing_enabled");
```

Comment:
To reduce the confusion mentioned in the OP, maybe move those two to a group called "Rotation Smoothing"?

(Also: `smoothing_rotation_enabled` should toggle internal process too, if it is to be usable without `Smoothing` enabled)

Reply:
Another group for Rotation Smoothing would work. 

Should the "Smoothing" group be renamed to "Follow Smoothing" (or something similar) to help clarify the distinction between the two smooth groups?

Reply:
> (Also: `smoothing_rotation_enabled` should toggle internal process too, if it is to be usable without `smoothing_enabled`)

What will be the best way to handle internal_process toggling when one smoothing is > 0 and the other is not?

For example if follow smoothing is > 0 but rotation smoothing is <= 0, is it valid to set internal process to true?

Reply:
If either smoothing or rotation smoothing is enabled, you need process on. So, doesn't matter which one of the two requires it, just one of them is enough.

Reply:
@bojidar-bg 
> If either smoothing or rotation smoothing is enabled, you need process on. So, doesn't matter which one of the two requires it, just one of them is enough.

Could you clarify why we enable internal process for the given condition
```
if (smoothing > 0 && !(is_inside_tree() && Engine::get_singleton()->is_editor_hint()))
      set_process_internal(true);
```
So it can be set true only smoothing has a value and is not inside the tree and not in editor.